frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -600,6 +600,13 @@ const HostDetailsPage = ({
   const isMdmUnenrolled =
     host?.mdm.enrollment_status === "Off" || !host?.mdm.enrollment_status;
 
+  const showDiskEncryptionUserActionRequired =
+    config?.mdm.enabled_and_configured &&
+    host?.mdm.name === "Fleet" &&
+    host?.mdm.macos_settings.disk_encryption === "action_required";
+
+  const isAppleBmTermsExpired = config?.mdm?.apple_bm_terms_expired;
```

Comment:
Should `isAppleBmTermsExpired` be referenced somewhere in the new changes? If so, I'm missing it.

Reply:
Woops, I'll clean that up